packages/python-kombu/python-kombu.spec

```diff
@@ -14,7 +14,9 @@ Source0:        https://github.com/celery/kombu/archive/v%{version}.tar.gz#/%{sr
 # This can be removed in 4.0.3+
 Patch0: qpid-transport-works-with-celery-4.patch
 Patch1: 785.patch
- 
+# This can be removed in 4.2+
```

Comment:
https://github.com/celery/kombu/commit/4e28c0b839171ffddc8cbb43e0909adcd25f7e75 is already included in 4.1.0

Reply:
The comment isn't strictly correct, but the sources in this package are for 4.0.2.  We're likely going to skip over 4.1